sections/featured-blog.liquid

```diff
@@ -91,24 +91,46 @@
                 style="--animation-order: {{ forloop.index }};"
               {% endif %} 
             >
-              <div
-                class="blog-placeholder{% if settings.animations_reveal_on_scroll %} scroll-trigger animate--slide-in{% endif %}"
-                {% if settings.animations_reveal_on_scroll %}
-                  data-cascade
-                {% endif %}
-              >
-                {%- if section.settings.show_image == true -%}
-                  <div class="placeholder media media--hover-effect media--landscape">
-                    {{ placeholder_image | placeholder_svg_tag: 'placeholder-svg' }}
-                  </div>
-                {% endif %}
-                <div class="blog-placeholder__content">
-                  <h2>
-                    {{ 'sections.featured_blog.onboarding_title' | t }}
-                  </h2>
-                  <p class="rte-width">
-                    {{ 'sections.featured_blog.onboarding_content' | t }}
-                  </p>
+              <div class="article-card-wrapper card-wrapper{% if settings.animations_reveal_on_scroll %} scroll-trigger animate--slide-in{% endif %}">
+                <div class="card article-card card--{{ settings.blog_card_style }}
+                    {% if settings.blog_card_style == 'card' %} color-{{ settings.blog_card_color_scheme }} gradient{% endif %}
+                    {% if section.settings.show_image %} card--media{% else %} card--text{% endif %}"
+                   style=" {% if settings.blog_card_style == 'standard' %} --ratio-percent: 100%;{% elsif settings.blog_card_style == 'card' %} --ratio-percent: 0%; flex-direction: column;{% endif %}"
```

Comment:
Since we don't have the `ratio` classe anymore, you can remove the flex direction you have in here. It's already taken care of:
![](https://screenshot.click/22-18-q1gak-4c2e3.png)

```suggestion
                   style=" {% if settings.blog_card_style == 'standard' %} --ratio-percent: 100%;{% elsif settings.blog_card_style == 'card' %} --ratio-percent: 0%; flex-direction: column;{% endif %}"
```